### PR TITLE
Update exploring-kubernetes-cronjobs.md

### LINKED ISSUE #11783: [Docs] Add some spacing between image and text on tutorials
### Current State
In the pages mentioned below, the text and images have no space between them:
 - [Exploring Kubernetes CronJobs](https://docs.meshery.io/guides/tutorials/exploring-kubernetes-cronjobs)
 - [Exploring Kubernetes Pods ](https://docs.meshery.io/guides/tutorials/kubernetes-pods)

### Desired State
Add some space between the text and images, consistent with other pages.

image

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by priyagupta20044**
Added space between text and image

**Notes for Reviewers**

This PR fixes #11783 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 9b391be4c03c0a25f2b0ef35690eaf657d90f0ca at: https://66cc7bb554ef9a354fff3373--meshery-docs-preview.netlify.app

**Comment by marblom007** [comment]
Maybe @Alero-Awani can offer some feedback?

**Comment by leecalcote** [comment]
@SAHU-01 you might be able to as well.

**Review by Alero-Awani — CHANGES_REQUESTED** [review]

**Comment by amitamrutiya** [comment]
@priyagupta20044, refer to this [page](https://docs.meshery.io/guides/tutorials/kubernetes-configmaps-secrets) and try to implement the spacing between text and images as it has on that page.

**Comment by sangramrath** [comment]
Since there are reviews similar to the ones I have, I am adding mine as comment here:
- Numbering has changed
- Image alignment has moved out of the numbered list
Screenshot 2024-08-26 at 10 51 39 PM

File: docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md [hunk 1]
```diff
@@ -41,19 +41,25 @@ To deploy a web application on Meshery, follow these steps:
 1. Navigate to the Configuration menu in Meshery.
 2. Select "Designs" from the menu.
 3. Click on the "Import Design" button.
+   
 [![Configuration menu]({{site.baseurl}}/assets/img/meshery-design/configuration-menu-design-import.png)]({{site.baseurl}}/assets/img/meshery-design/configuration-menu-design-import.png)
```
Comment (sangramrath) [inline docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md:45]: Image has moved outside of the numbered list indentation.

File: docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md [hunk 2]
```diff
@@ -41,19 +41,25 @@ To deploy a web application on Meshery, follow these steps:
...
-4. Fill in the required details for your application or design, including the Design File Name, Design Type, and choose your preferred upload method:
+5. Fill in the required details for your application or design, including the Design File Name, Design Type, and choose your preferred upload method:
```
[the remark this answers is not in the document]
Comment (sangramrath) [reply docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md:46]: +1

**Comment by Ashparshp** [comment]
@priyagupta20044. let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

File: docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/tutorials/exploring-kubernetes-cronjobs.md:45]: Great feedback, @sangramrath 🙌

**Comment by leecalcote** [comment]
it's great to see this PR, @priyagupta20044. You're almost done...

**Comment by sudhanshutech** [comment]
@priyagupta20044 there are lots of feedback to incorporate. Are you still working in this?

**Pull request closed without merging**